src/components/listings/MasonryListItem.tsx

```diff
@@ -0,0 +1,42 @@
+import React, { useState } from 'react'
+import { ImageSourcePropType } from 'react-native'
+import { Image } from 'expo-image'
+
+interface MasonryListItemProps {
+  image: ImageSourcePropType
+  containerWidth: number
+}
+
+function MasonryListItem(props: MasonryListItemProps) {
+  const { image, containerWidth } = props
+
+  const blurhash =
```

Comment:
Can we make this a const outside the component please?

```
const BLUR_FALLBACK_HASH = '..'

function MasonryListItem(props: MasonryListItemProps) {
```

Reply:
👍